src/schedule/edf_schedule.c

```diff
@@ -124,12 +129,16 @@ static void schedule_edf_task(void *data, struct task *task, uint64_t start,
 
 	/* add task to the list */
 	list_item_append(&task->list, &edf_sch->list);
+	tracev_edf_sch("schedule_edf_task() %p", (long)task);
```

Comment:
Note: if your object has uid (a task has one), you may use also %s and provide task->uid to print the task name and uuid value in the log.